src/main/java/com/kernicpanel/RandomizerExtension.java

```diff
@@ -98,9 +101,9 @@ private NoArgsCallback randomName() {
 
       if (useDate.get()) {
         LocalDate dateObj = LocalDate.now();
-        DateTimeFormatter formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd");
+        DateTimeFormatter formatter = DateTimeFormatter.ofPattern(dateFormat.get());
```

Comment:
If dateFormat contains something like `hgfjhgfj` we get an `IllegalArgumentException`
What about this ?
```suggestion
        DateTimeFormatter formatter;
        try {
           formatter = DateTimeFormatter.ofPattern(dateFormat.get());
        } catch (IllegalArgumentException e) {
          formatter = DateTimeFormatter.ofPattern("yyyy-MM-dd_");
        }
```
Or maybe, we could make some check as soon as the user changes the setting, and alert him right away ?

Reply:
This is definitely better. I'll need to check the API if I there is a callback when the settings were changed and If there is a way to open a dialog.